src/app/containers/ChartbeatAnalytics/amp/index.jsx

```diff
@@ -1,17 +1,27 @@
 import React from 'react';
-import { string, number } from 'prop-types';
+import { string, number, bool } from 'prop-types';
 
-const chartbeatAmpConfigOptions = options => ({
-  vars: {
+const chartbeatAmpConfigOptions = options => {
+  const vars = {
     uid: options.chartbeatUID,
+    title: options.title,
     sections: options.sections,
     domain: options.domain,
     contentType: options.type,
-    idSync: {
+  };
+
+  if (options.hasReferrer) {
```

Comment:
Instead of these conditions, in the above object you can just do:
```
...(options.hasReferrer && { virtualReferrer: options.referrer })
...(options.hasCookie && { idSync: { bbc_hid: options.cookie }})
```